embedded_data/embedded_file_template.cpp

```diff
@@ -0,0 +1,79 @@
+#include <optional>
+#include <unordered_map>
+#include "date/embedded_data_file.h"
+namespace date
+{
+enum class Timezone_file_name
+{
+	//EditLocationFileNameEnum
```

Comment:
Formatting looks different on this file

Reply:
I've run format.sh on this file to bring it inline with RTC, although its reporting no changes. Fixed some minor spacing issues, manually, but am struggling to spot other formatting issues. Sorry if I'm being blind, could you highlight some of them in case I'm being daft!

Reply:
Without a clang-format file at the root, the format.sh script won't be able to get it in a way where it follows RTC exactly. The most obvious one here is the use of tabs instead of spaces. No biggie though, I just thought I'd point it out while we're here since the tabs on GitHub are exaggerated.

Reply:
Ahh, that makes sense 🤦 

Will do some fiddling to make sure the template gets properly formatted by the script. Thanks for the catch!